tests/RobotFramework/tests/azure/azure_telemetry.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-az subscribes the topics that are configured as az.topics
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Custom Teardown
+
+Test Tags           theme:mqtt    theme:az
+
+
+*** Test Cases ***
+
+Publish measurements to te measurement topic
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main///m/ '{"temperature": 10}'
+    Should Have MQTT Messages    az/messages/events/device:main/m/   message_contains="temperature"    date_from=${timestamp}   minimum=1    maximum=1
+
+Publish measurements to te measurement topic with measurement type
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main///m/test-type '{"temperature": 10}'    
+    Should Have MQTT Messages    az/messages/events/device:main/m/test-type   message_contains="temperature"    date_from=${timestamp}   minimum=1    maximum=1
+
+Publish service measurements to te measurement topic with measurement type
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/main/service/test_service/m/test-type '{"temperature": 10}'
+    Should Have MQTT Messages    az/messages/events/device:main:service:test_service/m/test-type   message_contains="temperature"    date_from=${timestamp}   minimum=1    maximum=1
+
+Publish child measurements to te measurement topic with measurement type
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    tedge mqtt pub te/device/child///m/test-type '{"temperature": 10}'
+    Should Have MQTT Messages    az/messages/events/device:child/m/test-type   message_contains="temperature"    date_from=${timestamp}   minimum=1    maximum=1
+
+
+Publish main device event to te event topic with event type
```

Comment:
Why not adding without event type test as well?

Reply:
added